packages/ffe-datepicker-react/src/calendar/MonthPicker.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+
+export default function MonthPicker(props) {
+    const { previousMonthHandler, nextMonthHandler, month } = props;
+
+    function left(evt) {
+        previousMonthHandler(evt);
+    }
+
+    function right(evt) {
+        nextMonthHandler(evt);
+    }
+
+    return (
+        <div className="ffe-calendar__monthpicker">
+            <div className="ffe-calendar__monthpicker-center">{month}</div>
```

Comment:
er dette bare en div semantiskt?

Reply:
Yes. Not strictly necessary I think. May be a better solution.

Reply:
I was just wondering if it was a label or something else?